packages/google_maps_flutter/google_maps_flutter_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.13.0
+
+* Add Advanced marker support
```

Comment:
Adds

Also needs a period at the end.

(Both per style guide linked from the PR checklist.)